pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -16,6 +16,22 @@ def is_large_tensor(tensor, threshold):
     return size > threshold
 
 
+def _is_stripped(tensor):
+    if len(tensor.external_data) == 0:
+        return False
+    external_value = None
+    for external_data in tensor.external_data:
+        if external_data.key == 'location':
+            external_value = external_data.value
+    if not external_value:
+        return False
+    try:
+        external_value_dict = json.loads(external_value)
+        return external_value_dict.get('type', '') == 'stripped'
+    except Exception:
+        return False
```

Comment:
Let's use the narrower exception.

```suggestion
    except ValueError:
        # Invalid JSON, indicating `external_value` contains a file path
        return False
```

Reply:
fixed